handler/handler.go

```diff
@@ -452,6 +452,18 @@ func (h *handler) Score(w http.ResponseWriter, r *http.Request) {
 		g.Round++
 	}
 
+	if g.Round >= 13 && yahtzee.ContainsFeature(g.Features, yahtzee.TheChance) {
+		for _, p := range g.Players {
+			s := 0
+			for _, v := range p.ScoreSheet {
+				s += v
+			}
+			if s == 5 {
+				p.ScoreSheet[yahtzee.TheChance] = 495
```

Comment:
just set the `yahtzee.Bonus`, then it will show up on the score board without any extra thing to do